internal/browser/README.md

```diff
@@ -0,0 +1 @@
+This is a fork of https://github.com/pkg/browser that allows changing stdout/stderr non-globally.
```

Comment:
have you considered having a formal fork live at github.com/tilt-dev/browser? 

i don't think the changes we want here would be unreasonable upstream

Reply:
I figured I'd try an upstream PR and then make a formal fork if that goes poorly, but I guess I need a fork from which to make a PR anyway.